master/buildbot/test/util/migration.py

```diff
@@ -82,5 +84,17 @@ def upgrade_thd(engine):
                 schema.runchange(version, change, 1)
         d.addCallback(lambda _: self.db.pool.do_with_engine(upgrade_thd))
 
+        def check_table_charsets(engine):
+            # charsets are only a problem for MySQL
+            if engine.dialect.name != 'mysql':
+                return
+            dbs = [r[0] for r in engine.execute("show tables")]
+            for tbl in dbs:
+                r = engine.execute("show create table %s" % tbl)
+                create_table = r.fetchone()[1]
+                self.assertIn('DEFAULT CHARSET=utf8', create_table,
```

Comment:
What if non-Buildbot tables will be in the DB? Or it's unsupported configuration?
Maybe some external tools (phpMyAdmin?) will create additional tables, for example to add additional metadata?